components/brave_rewards/resources/rewards_panel/lib/interfaces.ts

```diff
@@ -64,13 +64,18 @@ export interface Options {
   autoContributeAmounts: number[]
 }
 
+type RequestedView = 'rewards-tour' | 'brave-talk-opt-in'
+
 export interface HostState {
+  openTime: number
```

Comment:
do you need this since it's not currently being cached, or is this for future use?